mythril/laser/ethereum/svm.py

```diff
@@ -161,7 +161,8 @@ def execute_state(self, global_state):
                 self._execute_post_hook(op_code, [e.global_state])
 
                 new_global_states = self._end_message_call(return_global_state, global_state,
-                                                           revert_changes=False, return_data=transaction.return_data)
+                                                           revert_changes=False or e.revert,
```

Comment:
it might be good to rename the exception ‘e’ right?

Reply:
I'm not sure. In these cases I think that err or e is often used
To what would you want to rename it?

Reply:
Yes, but while looking at the PR, I had to go up to check what e was as i had no clue what it might be from the lines mentioned in github's diff .So in those cases it might help.for variable name something in lines of it being a transaction end signal might help.